scripts/add-funds.sh

```diff
@@ -6,4 +6,4 @@ address=$1
 
 curl localhost:5050/mint \
     -H "Content-Type: application/json" \
-    -d "{ \"address\": \"$address\", \"amount\": 9999999999999999999999999999, \"lite\": false }"
+    -d "{ \"address\": \"$address\", \"amount\": 9999999999999999999999999999, \"lite\": false }"
```

Comment:
Why did this change?

Reply:
And is it still useful without the docs?

Reply:
done :)